src/style.css

```diff
@@ -1,3 +1,88 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 60px;
+  padding-left: 50px;
+  padding-right: 50px;
```

Comment:
Don't fix container size (if there is no such a requirement). It's better to use padding inside child elements or use CSS properties like max-width to ensure the container size is flexible and content-driven.